content/master/getting-started/introduction.md

```diff
@@ -100,14 +100,14 @@ For example, an AWS Provider defines Kubernetes CRDs for AWS resources like EC2
 compute instances or S3 storage buckets.
 
 The Provider defines the Kubernetes API definition for the external resource.
-For example, the 
-[Upbound Provider AWS](https://marketplace.upbound.io/providers/upbound/provider-aws/)
+For example,  
+[provider-upjet-aws](https://github.com/crossplane-contrib/provider-upjet-aws)
 defines a 
-[`bucket`](https://marketplace.upbound.io/providers/upbound/provider-aws/v0.25.0/resources/s3.aws.upbound.io/Bucket/v1beta1) 
+[`bucket`](https://github.com/crossplane-contrib/provider-upjet-aws/blob/release-1.20/package/crds/s3.aws.upbound.io_buckets.yaml) 
 resource for creating and managing AWS S3 storage buckets. 
 
 In the `bucket` CRD is a
-[`spec.forProvider.region`](https://marketplace.upbound.io/providers/upbound/provider-aws/v0.25.0/resources/s3.aws.upbound.io/Bucket/v1beta1#doc:spec-forProvider-region)
+[`spec.forProvider.region`](https://github.com/crossplane-contrib/provider-upjet-aws/blob/d83ef906bd17a7b6df814310e2ed7d674e99b786/package/crds/s3.aws.upbound.io_buckets.yaml#L91)
```

Comment:
do you want to use a release tag instead of a commit hash here, so it's more clear when looking at it what version it points to?